### ISSUE TITLE
Training Outputs

### ISSUE THREAD

**Issue by jingieboy** [roboflow/rf-detr#397]
Hi there, just playing around with RF-DETR, notice when training, beside the number of epochs (e.g Epoch: [144] [280/365]). Was wondering where does the 365 number come from. And after every epoch, it prints something out like this:

<img width="563" height="252" alt="Image" src="https://github.com/user-attachments/assets/a7a02ddc-46a5-4431-a5cf-146256d67427" />

I am assuming this is eval on the validation set, was wondering what does it mean, why are there multiple precision and recall scores. Thanks!

**Comment by Borda** [comment]
Hey @jingieboy, thanks for the question! The `365` in `Epoch: [144,280/365]` is the number of training steps per epoch (i.e. `len(train_loader)`), not the total number of epochs — it's a PyTorch convention from the AverageMeter display.

This is general usage clarification rather than a bug or feature request, so I'll close this to keep the issue tracker tidy. If something about the output seems wrong rather than just unfamiliar, feel free to open a new issue with specifics!
